pkg/plugins/runtime/k8s/controllers/gatewayapi/http_route_status.go

```diff
@@ -30,7 +30,8 @@ func (r *HTTPRouteReconciler) updateStatus(ctx context.Context, route *gatewayap
 // mergeHTTPRouteStatus updates the route status with the list of conditions for
 // each parent ref by mutating the given HTTPRoute.
 func mergeHTTPRouteStatus(route *gatewayapi.HTTPRoute, parentConditions ParentConditions) {
-	var mergedStatuses []gatewayapi.RouteParentStatus
+	// we cannot set a `nil` list
+	mergedStatuses := []gatewayapi.RouteParentStatus{}
 	var previousStatuses []gatewayapi.RouteParentStatus
```

Comment:
Does it also need a change?

Reply:
No, this is just an accumulator, not used to patch k8s objects.